applications/serial_lte_modem/src/main.c

```diff
@@ -375,6 +445,7 @@ int start_execute(void)
 	k_work_queue_start(&slm_work_q, slm_wq_stack_area,
 			   K_THREAD_STACK_SIZEOF(slm_wq_stack_area),
 			   SLM_WQ_PRIORITY, NULL);
+	(void)nidd_auto_connect();
```

Comment:
There have to be #ifdefs around this function call

Reply:
#ifdef is inside this function.